tests/collator_test.py

```diff
@@ -4,30 +4,42 @@
 
 
 @pytest.mark.parametrize(
-    ["arch", "has_features", "has_target", "expected_separate_features"],
     [
-        ("feature_invariant_transformer", True, True, False),
-        ("feature_invariant_transformer", True, False, False),
-        ("lstm", True, True, False),
-        ("lstm", False, True, False),
-        ("lstm", True, False, False),
-        ("lstm", False, False, False),
-        ("pointer_generator_lstm", True, True, True),
-        ("pointer_generator_lstm", False, True, False),
-        ("pointer_generator_lstm", True, False, True),
-        ("pointer_generator_lstm", False, False, False),
-        ("transducer", True, True, True),
-        ("transducer", False, True, False),
-        ("transducer", True, False, True),
-        ("transducer", False, False, False),
-        ("transformer", True, True, False),
-        ("transformer", False, True, False),
-        ("transformer", True, False, False),
-        ("transformer", False, False, False),
+        "arch",
+        "has_features",
+        "has_target",
+        "expected_separate_features",
+        "max_src_length",
+        "max_tgt_length",
+    ],
+    [
+        ("feature_invariant_transformer", True, True, False, 128, 128),
```

Comment:
If you take my advice and make 128 the default for both, this won't be necessary, it occurs to me.